src/map/entities/mobentity.cpp

```diff
@@ -1468,3 +1468,8 @@ bool CMobEntity::OnAttack(CAttackState& state, action_t& action)
         return CBattleEntity::OnAttack(state, action);
     }
 }
+
+bool CMobEntity::isWideScannable()
+{
+    return !(getMobMod(MOBMOD_NO_WIDESCAN));
```

Comment:
Did you test this with `CBaseEntity::isWideScannable()` as part of the condition? By doing this, you're entirely bypassing that original logic for _all mobs_

Reply:
Yea so without the ! it defaults to showing no mobs on widescan -

Reply:
This was the snippet I gave you:
```cpp
bool CMobEntity::isWideScannable()
{
    return CBaseEntity::isWideScannable() && getMobMod(MOBMOD_NO_WIDESCAN);
}
```

Sure, you need to negate the `getMobMod`, but the whole point of `override` is that you're replacing and bypassing the original call. So if you don't include the original call with `CBaseEntity::isWideScannable()`, the ONLY condition being checked is the mobmod - so it'll be skipping whether they're alive, or their name is visible etc.

Reply:
> Yea so without the ! it defaults to showing no mobs on widescan -

Really coz it looks like you are not asking that state and returning if your condition is right but actually just returning your condition..

Reply:
> This was the snippet I gave you:
> 
> ```c++
> bool CMobEntity::isWideScannable()
> {
>     return CBaseEntity::isWideScannable() && getMobMod(MOBMOD_NO_WIDESCAN);
> }
> ```
> 
> Sure, you need to negate the `getMobMod`, but the whole point of `override` is that you're replacing and bypassing the original call. So if you don't include the original call with `CBaseEntity::isWideScannable()`, the ONLY condition being checked is the mobmod - so it'll be skipping whether they're alive, or their name is visible etc.

That snippet is gonna default to false and return nothing causing nothing to show on widescan since its an override.

Reply:
We've already talked about the negation of the `getMobMod` call, I'm talking about the omission of `CBaseEntity::isWideScannable()` , which is this:
```cpp

bool CBaseEntity::isWideScannable()
{
    return status != STATUS_TYPE::DISAPPEAR && !IsNameHidden() && !GetUntargetable();
}
``` 

Once the negation is in, what if the mob is set as untargettable, hidden, or disappear? They'll appear on widescan.

Reply:
What I'm asking for is this:
```cpp
bool CMobEntity::isWideScannable()
{
    return CBaseEntity::isWideScannable() && !getMobMod(MOBMOD_NO_WIDESCAN);
}
```

We do a very similar thing in NPCEntity:
```cpp
bool CNpcEntity::isWideScannable()
{
    return widescan == 1 && status == STATUS_NORMAL && CBaseEntity::isWideScannable();
}
```

Reply:
Ah yes, I see. Thank you :D

Reply:
I see what you mean also - thanks. I'll rebase when franks gets merged and make those adjustments. I uh, not too good at reading what's going on. Was trying to get do the right thing by PR it but didn't realize the second order effects. Thanks again.